src/gui/folderman.cpp

```diff
@@ -849,7 +849,7 @@ Folder *FolderMan::folderForPath(const QString &path)
     foreach(Folder* folder, this->map().values()) {
         const QString folderPath = folder->cleanPath()+QLatin1Char('/');
 
-        if(absolutePath.startsWith(folderPath)) {
+        if(absolutePath.startsWith(folderPath, Qt::CaseInsensitive)) {
```

Comment:
I doubt that Mac is case sensitive. It is case **preserving**, but */Users/kf/Buds/Spencer* is the same file as */Users/kf/buds/Spencer*, right?